packages/@react-aria/table/src/useTableColumnHeader.ts

```diff
@@ -40,30 +42,39 @@ interface ColumnHeaderAria {
  */
 export function useTableColumnHeader<T>(props: ColumnHeaderProps, state: TableState<T>, ref: RefObject<HTMLElement>): ColumnHeaderAria {
   let {node} = props;
+  let allowsSorting = node.props.allowsSorting;
   let {gridCellProps} = useGridCell(props, state, ref);
 
   let isSelectionCellDisabled = node.props.isSelectionCell && state.selectionManager.selectionMode === 'single';
   let {pressProps} = usePress({
-    isDisabled: !node.props.allowsSorting || isSelectionCellDisabled,
+    isDisabled: !allowsSorting || isSelectionCellDisabled,
     onPress() {
       state.sort(node.key);
     }
   });
 
   // Needed to pick up the focusable context, enabling things like Tooltips for example
   let {focusableProps} = useFocusable({}, ref);
-  let ariaSort: HTMLAttributes<HTMLElement>['aria-sort'] = null;
-  if (node.props.allowsSorting) {
-    ariaSort = state.sortDescriptor?.column === node.key ? state.sortDescriptor.direction : 'none';
+
+  let formatMessage = useMessageFormatter(intlMessages);
+  let sortDescription;
+  let isSortedColumn = state.sortDescriptor?.column === node.key;
+  let sortDirection = state.sortDescriptor?.direction;
+  if (allowsSorting) {
+    sortDescription = `${formatMessage('sortable')}`;
+    if (isSortedColumn && sortDirection) {
+      sortDescription = `${sortDescription}, ${formatMessage(sortDirection)}`;
+    }
   }
 
+  let descriptionProps = useDescription(sortDescription);
```

Comment:
one pro about using aria-describedby here instead of aria-label is that android TalkBack properly announces the column text

Reply:
`aria-describedby` sounds like the way to go.

Reply:
Was this only supposed to be added for talkback? I hear "sortable column" twice with VoiceOver

Reply:
Are you referring to when VoiceOver mentions "sortable column (column name)" then pauses and says "sortable column. You are currently in a cell etc"? If so that is just how `aria-descibedby` is announced by VoiceOver it seems

Reply:
Oh are you sure it's not because `aria-sort` is also on the column?

Reply:
Yep, removing `aria-sort` doesn't affect the aforementioned announcement. `aria-sort` only gets announced by VoiceOver if it has a value other than "none". If it has a value of `ascending/descending` the announcement changes to "sortable column (column name), sort (up/down)".